src/client/wieldmesh.cpp

```diff
@@ -688,7 +710,9 @@ scene::SMesh *getExtrudedMesh(ITextureSource *tsrc,
 	return mesh;
 }
 
-void postProcessNodeMesh(scene::SMesh *mesh, const ContentFeatures &f,
+// FIXME: this is weirdly inconsistent with what MapBlockMesh does.
+// also set_material is never true.
```

Comment:
Would you please be so nice to keep the previous description/comment from the header file? I think it's a good summary.

Reply:
it's still there are the very top